Modules/api_handler.py

```diff
@@ -80,3 +80,19 @@ def __init__(self, hostname: str, token: str=None, tls=False):
             await self._connection.send(data)
         else:
             await self._connection.send(json.dumps(data))
+
+    def construct_request(self, endpoint: (str, str), root_params: {} = {}, meta_params: {} = {}) -> str:
```

Comment:
The type hints here are a bit off, it needs to be `root_params: dict, meta_params: dict`